scripts/lint.py

```diff
@@ -342,6 +342,38 @@ def check_rule(self, ctx, rule):
         return False
 
 
+class FormatStringIncorrect(Lint):
+    name = "rule string format incorrect"
+
+    def check_rule(self, ctx, rule):
+        for line in rule.definition.splitlines():
+            # strip "- string: example # comment" -> "string: example # comment"
+            line = line.strip().lstrip("-").strip()
+            if not line.startswith("string:"):
+                continue
+            # strip "string: example # comment" -> "example # comment"
+            value = line.partition(" ")[2].strip()
+            # strip "example # comment" -> "example"
+            value = value.rpartition("#")[2].strip()
```

Comment:
quite the parsing here, testing using `capafmt` I see that ruamel keeps track of the quote chars, so maybe you can get access to the values through it more easily